src/app/community-list-page/community-list-service.ts

```diff
@@ -23,6 +23,7 @@ import { followLink } from '../shared/utils/follow-link-config.model';
 import { FlatNode } from './flat-node.model';
 import { ShowMoreFlatNode } from './show-more-flat-node.model';
 import { FindListOptions } from '../core/data/find-list-options.model';
+import { environment } from 'src/environments/environment';
```

Comment:
It's better to inject the environment using
```typescript
@Inject(APP_CONFIG) protected appConfig: AppConfig
```
in the constructor, and then use `this.appConfig.browseCommunities.communityListPageSize` later on. That makes it easier for us to mock the environment in tests.